packages/medusa/src/commands/develop.js

```diff
@@ -19,10 +18,8 @@ export default async function ({ port, directory }) {
   })
 
   const cliPath = path.join(directory, "node_modules", ".bin", "medusa")
-  let child = spawn(cliPath, [`start`, ...args], {
-    cwd: directory,
-    env: process.env,
-    stdio: ["pipe", process.stdout, process.stderr],
+  let child = fork(cliPath, [`start`, ...args], {
+    execArgv: ["--preserve-symlinks", ...process.execArgv],
```

Comment:
Suggestion: we shouldn’t pass the flag automatically, it can create unexpected behaviour with some libs and a user could use something else than link (e.g. pack or other tools)